### PR TITLE
Make files to follow the standard

### PR DESCRIPTION
**Pull request by riyafa**

### CODE REVIEW

**Review by pubudu91 — COMMENTED** [review]
Double check. Some renames have missed some of the `-`s

File: examples/http_to_websocket_upgrade/http_to_websocket_upgrade.bal [hunk 1]
```diff
@@ -46,11 +46,11 @@ service<http:Service> httpService bind servicEp {
...
-//Note: When a WebSocket upgrade path is defined in HTTP resource configuration.
-//- Without service configuration for WebSocketService default values are taken for sub protocols, idle timeout etc...
-//- If  WebSocketServiceConfig is defined without the path, sub protocols, idle timeout etc... can be configured.
-//- If path is defined in the WebSocketServiceConfig it shall be ignored.
-//- This service can also be bound to a different endpoint in which case the path configuration will become useful.
+// Note: When a WebSocket upgrade path is defined in HTTP resource configuration.
+// - Without service configuration for WebSocketService default values are taken for sub protocols, idle timeout etc...
+// - If  WebSocketServiceConfig is defined without the path, sub protocols, idle timeout etc... can be configured.
+// - If path is defined in the WebSocketServiceConfig it shall be ignored.
+// - This service can also be bound to a different endpoint in which case the path configuration will become useful.
```
Comment (irunika) [inline examples/http_to_websocket_upgrade/http_to_websocket_upgrade.bal]: We should not use future tense when documenting (See package documentation). I think that is valid here too. :)

File: examples/produces_consumes/produces_consumes.bal [hunk 2]
```diff
@@ -6,26 +6,26 @@ endpoint http:Listener infoServiceEP {
     port:9092
 };
 
-//Consumes and produces annotations that contain MIME types as an array of strings.
+// Consumes and produces annotations that contain MIME types as an array of strings.
 service<http:Service> infoService bind infoServiceEP {
```
Comment (irunika) [inline examples/produces_consumes/produces_consumes.bal]: We can say ```ballerina service<http:Service> infoService bind {port:9092} ``` so we can get rid of infoServiceEP.

**Comment by irunika** [comment]
Also please find places where we can put 
```service<http:SomeService> someService bind {port:somePort} ``` then we can get rid of some redundant ```endpoint``` codes.

File: examples/produces_consumes/produces_consumes.bal (hunk 2 above)
Comment (irunika) [reply examples/produces_consumes/produces_consumes.bal]: Please check other samples 
- https://github.com/ballerina-platform/ballerina-examples/blob/master/examples/http_caching_client/http_caching_client.bal
- https://github.com/ballerina-platform/ballerina-examples/blob/master/examples/http_circuit_breaker/http_circuit_breaker.bal

File: examples/http_redirects/http_redirects.bal [hunk 3]
```diff
@@ -3,14 +3,14 @@ import ballerina/http;
...
+    followRedirects: {enabled: true, maxCount: 5}
 };
 
-function main (string... args) {
+function main(string... args) {
 
-    //Send a GET request to the specified endpoint
-    var returnResult = clientEP -> get("/v2/59d590762700000a049cd694");
+    // Send a GET request to the specified endpoint
+    var returnResult = clientEP->get("/v2/59d590762700000a049cd694");
```
Comment (Bhashinee) [inline examples/http_redirects/http_redirects.bal]: We need to use 'error' instead of 'http:HttpConnectorError' and 'http:PayloadError'

**Review by irunika — APPROVED** [review]

**Pull request merged**